### PR TITLE
Digital Subscription Product Page - Product Block

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

This adds the "product block" (i.e. the second section) to the digital sub landing page.

[**Trello Card**](https://trello.com/c/8UBqJMTP/1532-product-block)

cc @JustinPinner 

## Changes

- Tweaked the `FeatureList` component to take a React `Node` as a heading, rather than just a `string`.
- Added a new `Heading` component to abstract creation of `h1-6` elements. This is used so that the `FeatureList` headings make semantic sense in the context of the page where they are used; good for a11y.
- Added two new images from the Grid, and a new circles-themed SVG.
- Created a new "product block" component for the digital sub landing page.

## Screenshots

Breakpoint | Screenshot
-|-
LeftCol/Wide | product-leftcol
Desktop | product-desktop
Tablet | product-tablet
Phablet | product-phablet
Mobile Landscape | product-mobileland
Mobile | product-mobile

### CODE REVIEW

**Comment by rupertbates** [comment]
The designs don't have a margin to the left of the dropline on tablet (the content is full width) whereas this does, @Amohkhan are you ok with this change?

File: assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss [hunk 1]
```diff
@@ -22,4 +22,212 @@
...
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+  }
+
+  .component-feature-list {
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+    list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#ff4e36"/></svg>');
```
Comment (rupertbates) [inline assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss]: Might be better to break this out into a separate svg file?

File: assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss [hunk 2]
```diff
@@ -22,4 +22,212 @@
...
+        left: -100px;
+      }
+
+      @include mq($from: leftCol) {
+        left: -135px;
+      }
+    }
+
+    .product-block__product-heading {
+      background-color: gu-colour(sport-garnett-media-main-1);
```
Comment (rupertbates) [inline assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss]: Maybe we should introduce a new colour here? If the sport colour changes we wouldn't necessarily want this to change along with it.

File: assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss [hunk 3]
```diff
@@ -22,4 +22,212 @@
...
+    }
+
+    .component-feature-list {
+      list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#00b2ff"/></svg>');
```
Comment (rupertbates) [inline assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss]: Same comment about separate svg

**Comment by Amohkhan** [comment]
@rupertbates It looks fine like this, if anyone wants to correct that we'd lose a few widows which would be good, but the difference is pretty insignificant from what I can see.

**Comment by Amohkhan** [comment]
@rupertbates Jamie just explained that to me, it's not ideal but seems like it's necessary for the other breakpoints and so I'm happy to keep it there.

File: assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss (hunk 2 above)
Comment (Ap0c) [reply assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss]: As discussed offline, we're going to look into giving the palette more support-related names.

File: assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss (hunk 1 above)
Comment (Ap0c) [reply assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss]: As discussed offline, once built this produces a [`data URI`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs).

**Comment by Ap0c** [comment]
@Amohkhan @rupertbates as discussed offline, this left border now comes in at desktop.

**Pull request merged**